test/box-luatest/gh_8714_set_null_to_xxx_uuid_test.lua

```diff
@@ -0,0 +1,82 @@
+local server = require('luatest.server')
+local t = require('luatest')
+
+local g = t.group()
+
+g.test_set_null_to_xxx_uuid = function()
+    g.server = server:new({
+        alias = 'master',
+        box_cfg = {
+            instance_uuid = box.NULL,
+            replicaset_uuid = box.NULL,
+        },
+    })
+    g.server:start()
+    g.server:exec(function()
+        local ok, err = pcall(box.cfg, {instance_uuid = box.NULL})
+        t.assert(ok)
+        t.assert_equals(err, nil)
+        ok, err = pcall(box.cfg, {replicaset_uuid = box.NULL})
+        t.assert(ok)
+        t.assert_equals(err, nil)
+    end)
+    g.server:stop()
```

Comment:
If the test fails, the server won't be stopped properly. You should use `before_test` to start the server and `after_test` to stop it, because `after_test` runs even on test failure.

Reply:
Thank you for the review! Fixed.